pm_muairss_read/test-data/clusters.txt

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
Can't comment on the files themselves, but you can delete `cluster-out.zip` and `clustout-dftb.zip` now